### DISCUSSION TITLE
Are the WinUI Community Calls canceled?

### DISCUSSION THREAD

**Discussion by llothar** [microsoft/microsoft-ui-xaml#8802] [General]
Last community call was in June. Is this just a summer break or a break down. 

With many members of the old team moved to increase the AI hype i fear that WinUI/Windows App SDK will get even worse support now. Please tell me can get one in September again.

**Comment by gautambjain** [comment]
In June they had announced that next call would be in September.

**Comment by kmgallahan** [comment]
You are both concerned about the calls but also didn't watch the last one where this was discussed 🤷‍♂️.

**Comment by ghord** [comment]
Now that we are in the middle of September, any plans for the call? I've been enjoying the progress in winui3 lately, and I've been eagerly waiting for open source news.

**Reply by DarranRowe** [reply]
There was a post in the June thread stating that it would be on the 27th September.

**Comment by zadjii-msft** [comment]
Yea, let's just close this now that the community call thread was posted over in #8884
